### DISCUSSION TITLE
pokemon armor idea

### DISCUSSION THREAD

**Discussion by TheUnknownAccount** [Pokecube-Development/Pokecube-Issues-and-Wiki#640] [Ideas]
this idea, would be like the saddle, where the armor would appear invisible, but it would give said pokemon extra defense, and you could add enchantments, for example "fire resistance" so then your pokemon would take less damage from fire attacks  and other fire related stuff(like lava), the slots could be added next to the saddle/item area, for example there would be 4 slots for 1 piece of armor(the armor would either be a new item called "poke armor" which would have different tier like iron diamond gold, and so on) or could just use normal minecraft armor, either way it would have 4 slots, head, torso, legs, and feet, so then you could just add a 2 by 2 box/area next to the saddle/held item area

**Comment by Thutmose** [comment]
Hmm, I can see about getting some framework code in for making armour affect the stats of the mob, it should be possible to do. After that I can look into how to equip such things
